Documentation/observability/metrics.rst

```diff
@@ -366,16 +366,19 @@ Name                                          Labels
-``clustermesh_remote_cluster_failures``         ``source_cluster``, ``source_node_name``, ``target_cluster`` Enabled    The total number of failures related to the remote cluster
-``clustermesh_remote_cluster_nodes``            ``source_cluster``, ``source_node_name``, ``target_cluster`` Enabled    The total number of nodes in the remote cluster
-``clustermesh_remote_cluster_last_failure_ts``  ``source_cluster``, ``source_node_name``, ``target_cluster`` Enabled    The timestamp of the last failure of the remote cluster
-``clustermesh_remote_cluster_readiness_status`` ``source_cluster``, ``source_node_name``, ``target_cluster`` Enabled    The readiness status of the remote cluster
-=============================================== ============================================================ ========== =================================================================
+================================================== ============================================================ ========== =================================================================
+Name                                               Labels                                                       Default    Description
+================================================== ============================================================ ========== =================================================================
+``clustermesh_global_services``                    ``source_cluster``, ``source_node_name``                     Enabled    The total number of global services in the cluster mesh
+``clustermesh_remote_clusters``                    ``source_cluster``, ``source_node_name``                     Enabled    The total number of remote clusters meshed with the local cluster
+``clustermesh_remote_cluster_failures``            ``source_cluster``, ``source_node_name``, ``target_cluster`` Enabled    The total number of failures related to the remote cluster
+``clustermesh_remote_cluster_nodes``               ``source_cluster``, ``source_node_name``, ``target_cluster`` Enabled    The total number of nodes in the remote cluster
+``clustermesh_remote_cluster_last_failure_ts``     ``source_cluster``, ``source_node_name``, ``target_cluster`` Enabled    The timestamp of the last failure of the remote cluster
+``clustermesh_remote_cluster_readiness_status``    ``source_cluster``, ``source_node_name``, ``target_cluster`` Enabled    The readiness status of the remote cluster
+``cilium_operator_serviceexport_info``             ``serviceexport``, ``namespace``                             Enabled    Information about ServiceExport in the local cluster
+``cilium_operator_serviceexport_status_condition`` ``serviceexport``, ``namespace``, ``condition``, ``status``  Enabled    Status Condition of ServiceExport in the local cluster
+``cilium_operator_serviceimport_info``             ``serviceimport``, ``namespace``                             Enabled    Information about ServiceImport in the local cluster
```

Comment:
Could you run these metrics through `promlint` as well?

I thought we had a metrics linter configured somewhere but I can't seem to find it right now. But `promlint` seems to be the typical one to use.